README.md

```diff
@@ -25,6 +41,256 @@ OPTIONAL SECTIONS:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+- Basic knowdlenge of HTML, CSS, javascript, webpack and the code editor of your choice.
+- Basic knowlege of the computer operating system.
+
+
+### Setup <a name="setup"></a>
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Usage <a name="usage"></a>
+
+To run the project, execute the following command: go to your local repository and double click index.html file to open in a bowser. 
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+
+### Deployment <a name="triangular_flag_on_post-deployment"></a>
+
+You can deploy this project using: github pages or any host of your choice
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Lengushuru Charles**
+
+- GitHub: [@githubhandle](https://github.com/lengushuru)
+- Twitter: [@twitterhandle](https://twitter.com/ngushuru)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/lengushuru-charles-a09371111/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **responsive for all devices**
+- [ ] **use css to add more asthetics**
+- [ ] **use javascript class**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+If you like this project please give it a ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+I would like to thank microverse for the oportunity to learn how to set up this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+- **how will i setup this project in local machine**
+
+  - run the following command in the terminal: git clone https://github.com/lengushuru/awsome-books.git
+
+
+- **which code editor should i use**
+
+  - You can use any editor of your choice but i recommend vs code.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
```

Comment:
- [OPTIONAL] You have duplicate of your readme. Kindly remove the duplicate. The top README is fine.